Bottom: eaa8b9d02cd1465be862b8e5ca8d53023e7c6115
Top:    fa1bf5396713f49dc4283ba95bba356711891325
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:54:31 +0000

Refresh of fix-poly_contourincontour-test

---

diff --git a/src/polygon1.c b/src/polygon1.c
index c3bc1c9..11e6146 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2274,7 +2274,6 @@ poly_ContourInContour (PLINE * poly, PLINE * inner)
         } while ((pt = pt->next) != &inner->head);
       return 1;
     }
-  }
   return 0;
 }
